### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by ragog**
## Affected Components
* [ ] Resources
* [ ] Tests
* [x] Docs
* [ ] Other

## Style
* [ ] Terraform code is formatted with `terraform fmt`
* [ ] Go code is formatted with `go fmt`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,10 @@
...
 ## Introduction
 
-This Terraform provider enables users to manage [Checkly](https://checklyhq.com) resources like checks. You can read a detailed tutorial and explanation of the Checkly Terraform provider here:
+This Terraform provider enables users to manage [Checkly](https://checklyhq.com) resources like checks. You can read detailed tutorials and explanations of the Checkly Terraform provider here:
 
-* [Managing Checkly checks with Terraform](https://blog.checklyhq.com/managing-checkly-checks-with-terraform/)
+1. [Managing Checkly checks with Terraform](https://blog.checklyhq.com/managing-checkly-checks-with-terraform/)
+2. [Scaling Puppeteer and Playwright on Checkly with Terraform](https://blog.checklyhq.com/scaling-puppeteer-playwright-on-checkly-with-terraform/)
 
 ## Supported resources
```
Comment (ragog) [inline README.md:22]: hey @kjda, is this section up-to-date? is something still missing for the resources which are not marked?

File: README.md [hunk 2]
```diff
@@ -26,261 +27,17 @@ This Terraform provider enables users to manage [Checkly](https://checklyhq.com)
...
-  group_order = 1
-}
-```
-
-The `group_order` attribute specifies in which order the checks will be executed: 1, 2, 3, etc.
-
-
+For documentation and example usage see:
+1. [Checkly's documentation](https://www.checklyhq.com/docs/integrations/terraform/).
+2. [docs/resources](https://github.com/checkly/terraform-provider-checkly/tree/master/docs/resources)
```
Comment (hlenke) [inline README.md:39]: @ragog I think docs stored in GH there are auto deployed to https://registry.terraform.io/providers/checkly/checkly/latest/docs 
Hence we should link to the registry.
BTW: the docs seem to be broken.
Screenshot 2020-12-29 at 19 36 12

File: README.md [hunk 3]
```diff
@@ -26,261 +27,17 @@ This Terraform provider enables users to manage [Checkly](https://checklyhq.com)
...
 
-
 ## Installing the provider
 
 Please refer to the [installation guide](https://github.com/checkly/terraform-provider-checkly/blob/master/docs/guides/installing-the-provider-and-getting-started.md)
 
 If you're still using Terraform 0.12 please refer to [terraform 0.12 documentation](https://github.com/checkly/terraform-provider-checkly/blob/master/docs/guides/support-for-terraform-0.12.md)
 
 ## Using the provider
-For documentation and example usage look into [docs/resources](https://github.com/checkly/terraform-provider-checkly/tree/master/docs/resources). More examples can be found here `test.tf`(https://github.com/checkly/terraform-provider-checkly/blob/master/test.tf).
```
Comment (hlenke) [inline README.md:37]: @ragog did you check if all this is documented in /master/docs/ ?

File: README.md (hunk 2 above)
Comment (ragog) [reply README.md:39]: @hlenke yep, let me change this to a draft and fix the missing points

File: README.md (hunk 1 above)
Comment (ragog) [reply README.md:22]: Sorry for being unclear - in the above I am referring to the "Supported resources" section.

**Review by hlenke — APPROVED** [review]
LGTM - quick question: aren't alert channels supported now? cc @tnolet @kjda

**Pull request merged**